### ISSUE TITLE
Login issues after upgrading to 0.19.2

### ISSUE THREAD

**Issue by poVoq** [LemmyNet/lemmy#4379]
### Requirements

- [X] This is a bug report, and if not, please post to https://lemmy.ml/c/lemmy_support instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this issue already exists.
- [X] It's a single bug. Do not report multiple bugs in one issue.
- [X] It's a frontend issue, not a backend issue; Otherwise please create an issue on the [backend repo](https://github.com/LemmyNet/lemmy) instead.

### Summary

After upgrading from 0.18.5 to 0.19.2 all users with a login cookie are locked out of the page with an error message and the only way to fix this is to manually delete all site cookies, which is a bit hard to explain to users when they can't access the site.

I tried re-generating the jwt_secret in the database to forcefully log out users, but this had the opposite effect with the error re-appearing for users that had already removed their cookies, which lets me assume the problem might be that the ui doesn't recognize that the jwt secret changed. This would be a regression as this worked previously.

### Steps to Reproduce

1. upgrade from 0.18.5 to 0.19.2
2. error appears

### Technical Details

Official docker containers. Other instance admins seem to have the same issue.

### Lemmy Instance Version

0.19.2

### Lemmy Instance URL

https://slrpnk.net

**Comment by Nutomic** [comment]
Correct this is caused by https://github.com/LemmyNet/lemmy/pull/4321 which was released with Lemmy 0.19.1. Moving this to backend repo. Its fixed by https://github.com/LemmyNet/lemmy/pull/4380.

**Comment by Nothing4You** [comment]
It is still technically a lemmy-ui issue, isn't it?
Or, if you want to keep this in the lemmy repo for addressing the regression, a separate issue should be created in lemmy-ui (if one doesn't already exist) to cover the case of invalid auth cookies?

**Comment by aeharding** [comment]
The main issue is with lemmy-ui imho, where an invalid token causes the UI to become completely stuck, with the user unable to re-login.

Relevant: https://github.com/LemmyNet/lemmy/issues/4319#issuecomment-1894168489

**Comment by Nothing4You** [comment]
I just tested the scenario described for hexbear, although without actually doing an upgrade, and I can't reproduce it.

I've spun up a new 0.19.1 instance from scratch, set the same `jwt` cookie described above, which resulted in the invalid login messages.
It did however not prevent me from logging in and once I did the errors were (expectedly) gone.

To check whether this is different on an upgraded instance, I deleted the instance again and spun up yet another one.
On this instance, I first deployed 0.18.5, and logged in, then updated to 0.19.1 and tried to login, which also had the same behavior of not showing any issues.
